src/app.js

```diff
@@ -29,8 +30,18 @@
   _.extend(spiderOakApp, {
     config: window.spiderOakMobile_config,     // Supplemented in initialize.
     ready: function() {
+      var soApp = window.spiderOakApp,
```

Comment:
I committed a change to use window.spiderOakApp instead of soApp, to be more explicit and more like other tests.